src/app/modules/offers/sale-offers/add-sale-offer-form/add-sale-offer-form.component.html

```diff
@@ -64,7 +50,38 @@
             Item condition is required.
           </mat-error>
         </app-select>
+
+                <app-select
+          label="Delivery Type"
+          formControlName="deliveryType"
+          [options]="deliveryTypeService.deliveryTypeOptions.value()">
+          <mat-error *ngIf="form.controls['deliveryType'].hasError('required')">
```

Comment:
Sam app select ma to już obsłużone: https://github.com/karolNiedziela/UniqueShitWeb/blob/master/src/app/shared/components/select/select.component.html -> linia 28